src/init.cpp

```diff
@@ -954,7 +969,7 @@ bool AppInit2(Config &config, thread_group &threadGroup)
     LOGA("Default data directory %s\n", GetDefaultDataDir().string());
     LOGA("Using data directory %s\n", strDataDir);
     LOGA("Using config file %s\n", GetConfigFile(GetArg("-conf", BITCOIN_CONF_FILENAME)).string());
-    LOGA("Using at most %i connections (%i file descriptors available)\n", nMaxConnections, nFD);
+    LOGA("Using at most %i connections\n", nMaxConnections, nFD);
```

Comment:
You forgot to remove the `nFD` parameter

Reply:
good catch.

shouldn't we raise an error at compile time in this cases. seems like the LOGA macro is silently accepting more than needed parameters.

Reply:
woops. fixed